src/main/java/uk/co/ramp/covid/simulation/place/CareHome.java

```diff
@@ -76,4 +101,18 @@ public void stopIsolating() { }
     public void reportDeath(DailyStats s) {
         s.incCareHomeDeaths();
     }
+
+    @Override
+    public double getTransP(Person infected, Person target) {
```

Comment:
Looks great. My only small comment here is that while a care home resident is asymptomatic they should still be able to infect other care home residents.

One other thing that is a recent development in my brain, is that after somebody becomes symptomatic they should still be infectious for 6 hours after they become symptomatic before they get quarantined from other care home residents. Does thta make sense?